src/guides/about-multiprotocol.html

```diff
@@ -55,12 +55,30 @@ <h2 id="resolve-channel-conflict">Resolve channel conflict</h2>
           <li>Re-pair your devices.</li>
```

Comment:
This is not really related to this change, but still worth changing I think:

```suggestion
          <li>Re-pair your Thread devices.</li>
```